Project-1/README.Giacomo.md

```diff
@@ -0,0 +1,322 @@
+AlexanderI = A,
+Chris = C
+
+Concept Assignments:
+PersonI = {J, M, A, C},
+MaleI = {J, A, C},
+FemaleI = {M},
+BrotherI = {J},
+AuntI = {M}
+
+Role Assignments:
+parentofI = {(J, C), (A, J), (A, M)}
+
+
+6. Explain the difference - using natural language - between the first-order prefixes:
+  ```
+  (a) ∃x∀y and ∀x∃y
+  The first formula reads "there is a x, such that for all ys". Any relation coming afterwards will be a relation between this one x and all other elements in the domain.
+  The second formula reads "for all xs, there is a y such that". Any relation coming afterwards will relate at least one element y to all elements x in the domain.
+  (b) ∃x∀y∃z and ∀x∃y∀z 
+  The first formula reads "there is an x, for all y, there is a z such that". Any relation coming afterwards will relate one element x of the domain to all elements y of the domain which also are related to at least one element z.
+  The second formula reads "for all xs, there is a y, such that for all zs". Any relation coming afterwards will relate all the element x of the domain to at least an element y in the domain, which in turn is related to all elements z in the domain. 
+  (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+  The first formula reads "for all xs, there is a y, such that for all zs, there is a w such that"
+  Any relation coming afterwards will relate all elements x in the domain with at least an element y, which in turn relates to all the elements z in the domain, which in turn are each related to at least one element w.
+``The second formula reads "there is an x, such that for all ys, there is a z, such that all w".
+  Any relation coming afterwards will relate one element x in the domain with all elements y, which are each related to an element y, which in turn relate to all the elements w in the domain.
+
+7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
+```
+∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+
+<img width="205" alt="7" src="https://user-images.githubusercontent.com/117365623/218544552-d4b55a2f-6529-4459-a7f1-8949af713f85.png">
+
+```
+ {(a,b)(b,a)(a,a)} 
+ The second statement is true in this model. There is an x that relates every y (a, related to a and b) There is an x that relates to a z (where a is x and b is z), all ys (a and b) are related to some z (a is related to a and b, b is related to a)
+ The first statement is not true. It is not true that all xs (a and b) are related to all zs (b is not related to a)
+A graph is provided in the attached file.
+
+8. Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
+  ```
+See attached images.
+
+  (a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  
+  <img width="292" alt="8-a" src="https://user-images.githubusercontent.com/117365623/218544428-355ea0a8-b36a-4be7-aadb-7f4d47c89b10.png">

```

Comment:
What justifies moving from a negated universal x in the first line to a negated universal y in the second?